src/test/resources/application.conf

```diff
@@ -17,9 +17,13 @@ env: local
 local {
   services {
     host: "http://localhost"
-    auth-login-stub {
-      port: 9949
-      productionRoute = "/auth-login-stub"
+    economic-crime-levy-registration-frontend {
+      port: 14000
+      productionRoute = ""
+    }
+    auth-login-stub-ret {
```

Comment:
same as above - this should be the name of the service

`economic-crime-levy-returns-frontend`